tdrs-frontend/.env.development

```diff
@@ -1,4 +1,7 @@
 REACT_APP_BACKEND_URL=http://localhost:8080/v1
+REACT_APP_TIMEOUT_TIME=6000
+REACT_APP_LOGOUT_TIME=10000
+REACT_APP_DEBOUNCE_TIME=1000
```

Comment:
These are the environment variables for changing the timeout times for user sessions.

Reply:
Gotcha! And looks like these are the testing values, and they'll be updated after merge, is that right?

Reply:
Exactly!